### ISSUE TITLE
Extensions needed for dashboard metrics

### ISSUE THREAD

**Issue by dotnetshadow** [marcoCasamento/Hangfire.Redis.StackExchange#39]
Hi there,

I noticed with the latest release 1.6.7.7, you have an extension method to use **.UseRedisStore()** which is nice.

The problem I'm facing now is that if I use this extension method then I'm not sure how I can read dashboardmetrics without passing in a storage and redefining the storage:
`GlobalConfiguration.Configuration.UseDashboardMetric(storage.GetDashboardMetricFromRedisInfo("Redis Version", RedisInfoKeys.redis_version));`

It would me nice if I was able to do:
`GlobalConfiguration.Configuration.UseDashboardMetric(RedisStorage.GetDashboardMetricFromRedisInfo("Redis Version", RedisInfoKeys.redis_version));`

I've actually seen this implemented in another library:
https://github.com/BoltR/Hangfire.Redis.StackExchange

**Comment by marcoCasamento** [comment]
Yep, that has been added by @codeyu, a fluent syntax for configuration really helps on readability.
Seems like a nice addition, would you mind a PR ?

**Comment by pieceofsummer** [comment]
I think `GetDashboardMetricFromRedisInfo()` should not be an instance method of `RedisStorage` at all. It can safely be made static, or even turn into an extension method on `IGlobalConfiguration`.
